commcare_connect/solicitations/management/commands/test_dashboard_links.py

```
@@ -0,0 +1,145 @@
+
+        # Get the user
+        try:
+            user = User.objects.get(email=user_email)
+        except User.DoesNotExist:
+            self.stdout.write(self.style.ERROR(f"User {user_email} not found. Run UAT data script first."))
+            return
+
+        # Temporarily add testserver to ALLOWED_HOSTS for testing
+        current_allowed_hosts = list(settings.ALLOWED_HOSTS)
+        if "testserver" not in current_allowed_hosts:
+            settings.ALLOWED_HOSTS = current_allowed_hosts + ["testserver"]
```

Ruff (0.14.3)

Consider `[*current_allowed_hosts, "testserver"]` instead of concatenation

Replace with `[*current_allowed_hosts, "testserver"]`

(RUF005)